common/log.h

```diff
@@ -124,6 +124,4 @@ extern int isInit;
 #define YAMI_LOG_INFO 0x3
 #define YAMI_LOG_DEBUG 0x4
 
-void yamiTraceInit();
```

Comment:
it is in libyami git repo：https://github.com/01org/libyami/pull/522/commits/b707dbf70a9de0b0a431231ee86ada43640c71fd#diff-46169994b69f9b32ab65879fa86ba48bR44

it is inconvenient to split libyami / libyami-utils